cypress/support/functions.ts

```diff
@@ -1172,6 +1172,46 @@ Cypress.Commands.add('bindServiceFromSevicesPage', ({ appName, serviceName, bind
   }
 });
 
+Cypress.Commands.add('createServiceAndBindOneStep', ({ serviceName, catalogType, appName }) => {
+  cy.get('.accordion.package.depth-0.has-children', { timeout: 20000 }).contains('Services').click();
+  cy.clickButton('Create');
+  cy.typeValue({ label: 'Name', value: serviceName });
+  cy.get('input[placeholder="Select the type of Service to create"].vs__search').click();
+  cy.contains(catalogType).click();
+  // Verify selected catalog service is selected
+  cy.get('span.vs__selected').eq(1).should('contain', catalogType)
+  // Open "Bind to Applications" dropdown and bind service to app
+  cy.get('.v-select.inline.vs--multiple').click();
+  cy.contains(appName).should('be.visible').click();
+  // Click on create button
+  cy.clickButton('Create');
+  cy.get('.icon.icon-lg.icon-spinner.icon-spin', { timeout: 60000 }).contains('Creating...').should('not.exist');
+  // Confirm bound application after main instance page redirection
+  cy.contains('tr.main-row', serviceName, { timeout: 45000 }).within(() => {
+    cy.get('td[data-testid]', { timeout: 45000 }).eq(4).contains(appName).should('be.visible');
+  });
+});
+
+Cypress.Commands.add('countAndVerifyElements', ({ locator, numberRowsOrColumns, text1, text2, arrayOfElements = false }) => {
+
+  cy.get(locator).each((item, index, list) => {
+    expect(list).to.have.length(numberRowsOrColumns)
+
+    // This is in case several elements are to be checked per index
+    // as text1[elem-1, elem-2, elem-3,...], text2[elem-a, elem-b, elem-c,...]
+    if (arrayOfElements) {
```

Comment:
Wouldn't be possible, but only if not complicated, to do a detection if `text1` and `text2` are either `strings` or `arrays of strings`? Then we could simplify this logic block and also we would save one argument for the function itself.

Another possibility would be to drop support for the arrays as it is not used anywhere...

Reply:
yes. will do. Thanks